k8s/postgresql/README.md

```diff
@@ -283,15 +283,15 @@ Forward the port locally:
 ```shell
 kubectl port-forward \
   --namespace "${NAMESPACE}" \
-  "${APP_INSTANCE_NAME}-postgresql-0" 5432
+  "${APP_INSTANCE_NAME}-postgresql-0" 15432:5432
```

Comment:
Are you trying to forward port `5432` to `15432`?

I think it should be `5432:15432` instead.